helpers/hashimporter.rb

```diff
@@ -194,6 +194,22 @@ def importHashSalt(hash, hashfile_id, type)
   updateHashfileHashes(@hash_id.id.to_i, 'null', hashfile_id)
 end
 
+def importUserHashSalt(hash, hashfile_id, type)
+  data = hash.split(':', 2)
```

Comment:
Splits into two parts. Originally I split this without the '2' argument, so it split into three parts and then had to be concatenated back together again. This is neater.